wctc4xxp: from_timer is undefined

`from_timer` was renamed to `timer_container_of` in 6.16: https://github.com/torvalds/linux/commit/41cb08555c4164996d67c78b3bf1c658075b75f1

```
  LD [M]  wct4xxp/wct4xxp.o
  CC [M]  wctc4xxp/base.o
wctc4xxp/base.c: In function 'wctc4xxp_watchdog':
wctc4xxp/base.c:3657:28: error: implicit declaration of function 'from_timer'; did you mean 'mod_timer'? [-Werror=implicit-function-declaration]
 3657 |         struct wcdte *wc = from_timer(wc, timer, watchdog);
      |                            ^~~~~~~~~~
      |                            mod_timer
wctc4xxp/base.c:3657:50: error: 'watchdog' undeclared (first use in this function); did you mean 'watch'?
 3657 |         struct wcdte *wc = from_timer(wc, timer, watchdog);
      |                                                  ^~~~~~~~
      |                                                  watch
wctc4xxp/base.c:3657:50: note: each undeclared identifier is reported only once for each function it appears in
cc1: all warnings being treated as errors
make[5]: *** [/usr/src/linux/scripts/Makefile.build:287: wctc4xxp/base.o] Error 1
make[4]: *** [/usr/src/linux/scripts/Makefile.build:554: wctc4xxp] Error 2
make[3]: *** [/usr/src/linux/Makefile:2003: .] Error 2
make[2]: *** [/usr/src/linux/Makefile:248: __sub-make] Error 2
make[2]: Leaving directory '/usr/src/dahdi-linux-3.4.0/drivers/dahdi'
make[1]: *** [Makefile:248: __sub-make] Error 2
make[1]: Leaving directory '/usr/src/linux'
make: *** [Makefile:74: modules] Error 2
```

--- a/include/dahdi/kernel.h
+++ b/include/dahdi/kernel.h
@@ -58,6 +58,10 @@
 
 #include <linux/poll.h>
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 16, 0)
+#define from_timer timer_container_of
+#endif
+
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 1, 0)
 #define netif_napi_add netif_napi_add_weight
 #elif defined(RHEL_RELEASE_VERSION) && defined(RHEL_RELEASE_CODE)

